src/devtools/client/inspector/markup/components/MarkupApp.tsx

```diff
@@ -33,15 +33,10 @@ function MarkupApp(props: PropsFromRedux & { inspector: Inspector }) {
             <input
               id="inspector-searchbox"
               className="devtools-searchinput"
-              type="search"
+              type="input"
               placeholder="Search HTML"
+              autoComplete="off"
```

Comment:
Chrome still renders "x" for `type="search"` and has browser-specific CSS selectors to potentially style it slightly, thus the change to `"input"`. But then suggestions appear, so I've disabled those.